src/sentry/static/sentry/app/components/events/interfaces/debugMeta/index.tsx

```diff
@@ -140,7 +140,9 @@ class DebugMeta extends React.PureComponent<Props, State> {
   filterImage(image: Image) {
     const {showUnused, filter} = this.state;
 
-    if (!filter || filter.length < MIN_FILTER_LEN) {
+    const searchTerm = filter.toLowerCase();
+
+    if (!searchTerm || searchTerm.length < MIN_FILTER_LEN) {
```

Comment:
This condition is redundant now, given that the above can never be `null` or it will error. The filter *should* never be `null`, but it could be when `scrollToImage` is invoked in the frame with a `null` instruction address. Not sure if you want to protect against that.

```suggestion
    if (searchTerm.length < MIN_FILTER_LEN) {
```

Reply:
good catch...thanks!!